docs/chapters/04-1.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Linear Algebra and Convolutions
+authors: Yuchi Ge, Anshan He, Shuting Gu, and Weiyang Wen
+---
+
+## [Linear Algebra Review](0:00:00-0:21:26)
+
+This part is a recap of basic linear algebra in the context of neural networks. We start with a simple hidden layer $\vec{h}$:
+$$
+\vec{h} = f(\vec{z})
+$$
```

Comment:
You need to add an empty line before and after a centered equation, or it won't be rendered at all.
Here and all over the place.